sap-abap/services/textract/zcl_aws1_tex_actions.clas.abap

```diff
@@ -51,69 +51,66 @@ CLASS ZCL_AWS1_TEX_ACTIONS IMPLEMENTATION.
 
   METHOD analyze_document.
 
-    CONSTANTS: cv_pfl TYPE /aws1/rt_profile_id VALUE 'ZCODE_DEMO'.
+    CONSTANTS cv_pfl TYPE /aws1/rt_profile_id VALUE 'ZCODE_DEMO'.
 
     DATA(lo_session) = /aws1/cl_rt_session_aws=>create( cv_pfl ).
-    DATA(lo_tex) = /aws1/cl_tex_factory=>create( lo_session ).
-
+    DATA(lo_tex) = /aws1/cl_tex_factory=>create(
+      io_session = lo_session
+      iv_region = 'us-east-1' ).
```

Comment:
Typically we let the region be specified in the config file stored in ~/.aws and don't specify it in the code, unless it's required for some reason.